### DISCUSSION TITLE
CORE DDF POC Sprint Review 2023-11-20

### DISCUSSION THREAD

**Discussion by DianeWold** [cdisc-org/ddf-core-poc#62] [Show and tell]
UML Diagram marked up to show classes for selected rules.
![USDM_UML_2 5_annotated](https://github.com/cdisc-org/ddf-core-poc/assets/149505078/b560eba2-5636-4202-93af-28c15da67609)

**Comment by DianeWold** [comment]
When I look at the png of the marked-up UML diagram I attached above, I see three pictures.  Somehow attaching the file to this discussion allows only one of those pictures to be seen.  I don't know why.

**Comment by rbakercdisc1** [comment]
Topics in the UML document -
1. BiomedicalConceptProperty Entity with bcPropertyRequired:Boolean, bcPropertyEnabled:Boolean
2. Activity Entity with activityIsConditional:Boolean, activityIsConditionalReason:String (0..1)
3. Encounter Entity with name:String, description:String (0..1)
4. Timing Entity with timingWindow:String (0..1), timingWindowLower:String(0..1),timingWindowUpper:String(0..1)

**Comment by rbakercdisc1** [comment]
DDF00009 covers isConditional must have isConditionalReason
DDF00006 covers If any part of a timing window is specified then all parts should be specified
